README.md

```diff
@@ -1 +1,64 @@
-# first-capstone-summit
\ No newline at end of file
+# html-css-js-capstone-project
+
+This is my final project for HTML & CSS MODULE from the microverse curriculum
+
+<p align="center">
+  <img src="./img/screenshotmobile.png"/>
+</p>
+
+<hr>
+
+<p align="center">
+  <img src="./img/screenshotdesktop.png"/>
+</p>
```

Comment:
- [ ] The screenshot images are broken. I would recommend fixing that so your project is documented professionally.